### PR TITLE
pH7-Internationalization-Italian

### PR DESCRIPTION
**Pull request by DEATHJ0KER**
Fixed Files Italian Localization in the Italian language

[it_IT.zip](https://github.com/pH7Software/pH7-Internationalization/files/2922432/it_IT.zip)

### CODE REVIEW

File: _protected/app/langs/it_IT/config/config.ini [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[language]
+lang = it
+dir = ltr
+charset = "utf-8"
+
+[language.information]
+name = Italiano
+Author = "Valentino Decaroli"
```
Comment (pH-7) [inline _protected/app/langs/it_IT/config/config.ini:8]: Has to be lowercase only

**Review by pH-7 — CHANGES_REQUESTED** [review]

File: _protected/app/langs/it_IT/config/config.ini (hunk 1 above)
Comment (DEATHJ0KER) [reply _protected/app/langs/it_IT/config/config.ini:8]: `[language.information]
name = Italiano
author = "Valentino Decaroli"`

Author <--- author
The same error is present in the Spanish translation:

_protected/app/langs/es_ES/config/config.ini

With all the sample files I had available, I used the only file that has the syntax error… 🥇 LOL³ :)
A new update is needed regarding the pH7-Internationalization language pack.

Thank you Pierre for your attention

**Comment by DEATHJ0KER** [comment]
Updated version.
The new file that you made the changes, installing it returns the error 500 to the server.
The file uses UTF-8-BOM decoding.
I attach the correct file config.ini of the Italian and Spanish languages ​​save as UTF8 WITHOUT BOM


[pH7-Internationalization-master.zip](https://github.com/pH7Software/pH7-Internationalization/files/3000313/pH7-Internationalization-master.zip)

**Comment by Aiaboseacabo** [comment]
Same to me, all languages work well except italian: HTTP ERROR 500

Can you fix this bug?

**Comment by DEATHJ0KER** [comment]
Some spaces removed:
_protected/app/langs/it_IT/LC_MESSAGES/global.mo
[it_IT.zip](https://github.com/pH7Software/pH7-Internationalization/files/3132235/it_IT.zip)

no errors found:
_protected/app/langs/it_IT/config/config.ini 
_protected/app/langs/it_IT/language.php

carried out tests on: https://www.vxd.mobi/dating/

**Comment by Aiaboseacabo** [comment]
@DEATHJ0KER Thanks, now works!!!

**Comment by pH-7** [comment]
> Some spaces removed:
> _protected/app/langs/it_IT/LC_MESSAGES/global.mo
> [it_IT.zip](https://github.com/pH7Software/pH7-Internationalization/files/3132235/it_IT.zip)
> 
> no errors found:
> _protected/app/langs/it_IT/config/config.ini
> _protected/app/langs/it_IT/language.php
> 
> carried out tests on: https://www.vxd.mobi/dating/

Thanks a lot @DEATHJ0KER!
I really appreciate you update concerning the Italian translation. I will update it from the repo.

**Comment by pH-7** [comment]
The Italian translation has now been updated :tada: 

Thanks again @DEATHJ0KER! :100:

**Pull request closed without merging**